### ISSUE TITLE
Does not support public repo action paths

### ISSUE THREAD

**Issue by Nuru** [nektos/act#110]
According to the [official docs](https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#referencing-an-action-from-a-public-repository) the syntax for referring to an action from a public repo is
`{owner}/{repo}@{ref}` or `{owner}/{repo}/{path}@{ref}`.

`act` does not support the `{owner}/{repo}/{path}@{ref}` syntax. 

Given workflow `path-example.yml`
```yaml
name: Path Example
on: pull_request

jobs:
  example:
    runs-on: ubuntu-latest
    steps:
    - uses: cloudposse/actions/github/slash-command-dispatch@0.14.0
```

`act` treats the path as part of the repo name, not a path within it

```
 $ act
[Path Example/example] 🚀  Start image=node:12.6-buster-slim
[Path Example/example]   🐳  docker run image=node:12.6-buster-slim entrypoint=["/usr/bin/tail" "-f" "/dev/null"] cmd=[]
[Path Example/example]   🐳  docker cp src=/Users/nuru/dev/example/. dst=/github/workspace
[Path Example/example] ⭐  Run cloudposse/actions/github/slash-command-dispatch@0.14.0
[Path Example/example]   ☁  git clone 'https://github.com/cloudposse/actions' # ref=0.14.0
[Path Example/example]   ❌  Failure - cloudposse/actions/github/slash-command-dispatch@0.14.0
Error: open /Users/nuru/.cache/act/cloudposse-actions-github-slash-command-dispatch@0.14.0/action.yaml: no such file or directory
```

**Comment by cplee** [comment]
fixed on master, will be in next release.

**Comment by cplee** [comment]
available in `v0.2.5`
